tests/app/main/views/test_suppliers.py

```diff
@@ -1541,16 +1463,10 @@ def test_data_controller_role_can_update_registered_company_name(self, from_decl
         self.data_api_client.update_supplier.assert_called_once_with(
             1000, {'registeredName': "Something New"}, "test@example.com"
         )
-        assert self.data_api_client.update_supplier_declaration.call_args_list == ([
-            mock.call(
-                1000, 'g-cloud-11',
-                {'supplierRegisteredName': "Something New"}, "test@example.com"
-            )
-        ] if from_declaration else [])
+        assert self.data_api_client.update_supplier_declaration.call_args_list == []
```

Comment:
I don't think we need to assert this non-call if the code being tested doesn't use the method anywhere. More stuff to delete 😸

Reply:
👍

Reply:
👍